### DISCUSSION TITLE
Documentation requested about the functional design

### DISCUSSION THREAD

**Discussion by EstherHageraats** [AmpersandTarski/Ampersand#1243] [Q&A]
It will be a lot of work to do reverse engineering on this, while someone probably knows the answers.
Here are some of the questions I have:
- How are section headings defined? I have an adl-file that results in chapter 2.1 Loose ends and another that results in chapter 2.1 Define Model.
- One model results in diagrams and another does not. Having a population does not make a difference. So what is needed to get a diagram.
- Chapter 5 data structure has been emtpy except the first three introductory lines in all my generated design. What is needed to fill this section?
- Section 4 has in the introduction a reference to a glossary that does not exist.

**Comment by hanjoosten** [comment]
Well, not only reverse engineering. There still is quite some work to do to get a perfect document from the generator. Each chapter is ment for different audiences. Chapter two is meant for business people. It does not contain any IT-speak. That chapter generally works well, if you have purposes and meanings documented in your script. There are still tons of undocumented features as well as features that can be implemented better in the generator.

**Comment by sjcjoosten** [comment]
Can this issue be rephrased as a (list of) task(s)? Han?

If not, please answer the questions and close this issue, no need to keep it open.
